timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -121,7 +125,7 @@ def __init__(
         self.quantile_levels = quantile_levels or kwargs.get(
             "quantiles", [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]
         )
-
+        self.early_stopping_patience = early_stopping_patience
```

Comment:
predictors/learners should not have this as an instance variable as this can change from one `fit` call to another.

Reply:
Changed.